include/verilated_timing.h

```diff
@@ -0,0 +1,369 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+//
+// Code available from: https://verilator.org
+//
+// Copyright 2003-2022 by Wilson Snyder. This program is free software; you can
+// redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+#ifndef VERILATOR_VERILATED_TIMING_H_
+#define VERILATOR_VERILATED_TIMING_H_
+
+// Dynamic scheduling-related includes
+#include <functional>
+#include <queue>
+#include <unordered_map>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+// Some preprocessor magic to support both Clang and GCC coroutines with both libc++ and stdlibc++
+#ifdef __clang__
+#ifdef _LIBCPP_VERSION
+// Using libc++, coroutine library is in std::experimental
+#include <experimental/coroutine>
+namespace std {
+// Bring std::experimental into the std namespace
+using namespace experimental;
+}  // namespace std
+#else
+// Using stdlibc++, coroutine library is in std namespace
+#define __cpp_impl_coroutine 1  // clang doesn't define this, but it's needed in <coroutine>
+#include <coroutine>
+namespace std {
+// Bring coroutine library into std::experimental, as clang expects it to be there
+namespace experimental {
+using namespace std;
+}
+}  // namespace std
+#endif
+#else  // Not clang
+#include <coroutine>
+#endif
+
+#include "verilated.h"
```

Comment:
`verilated.h` seems unchanged, what is in there that needs the preprocessor magic above?

Reply:
It doesn't need it, I put it there so `verilated_timing.h` can serve as replacement for `verilated.h`, as the `#error` about including from `verilated.h` isn't there anymore. It also plays nicely with language servers/ctags etc. The generated syms file includes `verilated_timing.h` instead of `verilated.h`.

I can roll this back, I'm not particularly attached to this idea.

Reply:
Reverted it, as it wars confusing. Now I do it the same way as DPI.